util/stopwaiter/stopwaiter.go

```diff
@@ -102,7 +102,7 @@ func (s *StopWaiterSafe) StopOnly() {
 	s.stopped = true
 }
 
-// Stopping multiple times, even before start, will work
+// StopAndWait is stopping multiple times, even before start, will work
```

Comment:
```suggestion
// StopAndWait may be called multiple times, even before start.
```

I dunno, this didn't read well before. What do you think?